lib/jackalope/session.ex

```diff
@@ -43,6 +43,13 @@ defmodule Jackalope.Session do
     GenServer.cast(__MODULE__, {:report_connection_status, status})
   end
 
+  @spec connection_status() :: :online | :offline
+  def connection_status() do
+    GenServer.call(__MODULE__, :connection_status)
+  catch
+    _, _ -> false
```

Comment:
```suggestion
    _, _ -> :offline
```

This should prob be `:offline` to match the spec

Reply:
updated.